Docker/Dockerfile

```
@@ -0,0 +1,64 @@
+
+## Install additional R packages that need source compilation
+## These are installed via R because they need specific versions
+RUN Rscript -e "install.packages('terra', repos='http://cran.us.r-project.org', type='source')"
+RUN Rscript -e "install.packages(c('assertthat', 'slam', 'RcppArmadillo','codetools','BiocManager'), repos='http://cran.us.r-project.org', type='source')"
+RUN Rscript -e "install.packages('https://cran.r-project.org/src/contrib/Archive/prioritizr/prioritizr_8.0.4.tar.gz', repos=NULL, type='source')"
+RUN Rscript -e "install.packages('https://cran.r-project.org/src/contrib/symphony_0.1.1.tar.gz', repos=NULL, type='source')"
+RUN Rscript -e "BiocManager::install('lpsymphony')"
+
+
+## Add SyncroSim license files
+ADD Docker/ModelRunService_license_file.lic /home
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)